core/computed/metrics/responsiveness.js

```diff
@@ -88,8 +88,12 @@ class Responsiveness {
       return evt.name === 'EventTiming' && evt.ph !== 'e';
     });
 
-    if (candidates.length && !candidates.some(candidate => candidate.args.data?.frame)) {
```

Comment:
Cool I left a comment about this. lemme know if that captures what you're tellin me

Reply:
Comment is good, but I meant we don't want the `candidate.args.frame` events at all because they have the old, incorrect timing data, so basically want to revert all the code changes in this file :)

(though @connorjclark's suggestion of `!some` -> `every` is good because of the explicit length check right before that)